docs/src/main/sphinx/client/cli.rst

```diff
@@ -161,10 +161,19 @@ mode:
       EMACS editors. Defaults to ``EMACS``.
   * - ``--http-proxy``
     - Configures the URL of the HTTP proxy to connect to Trino.
+  * - ``--history-file``
+    - Path to the :ref:`history file <cli-history>`. Defaults to ``~/.trino_history``.
   * - ``--network-logging``
     - Configures the level of detail provided for network logging of the CLI.
       Defaults to ``NONE``, other options are ``BASIC``, ``HEADERS``, or
       ``BODY``.
+  * - ``--output-format-interactive=<format>``
```

Comment:
It might be a good idea to add this to the example in the `Configuration file` section since setting it to `AUTO` is useful.